node_tree.py

```diff
@@ -307,6 +478,46 @@ def draw_color(self, context, node):
         return self.nodule_color
 
 
+class SvLinkMatrix(bpy.types.Operator):
```

Comment:
It would be cool if the spawned nodes can be offset in a smart way to not overlap with each other. Perhaps add slightly different offset depending on the index of the input socket ?